widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetController.kt

```diff
@@ -72,7 +72,7 @@ internal class EntryWidgetController(
                     if (allMedias.contains(Engagement.MediaType.VIDEO)) add(EntryWidgetContract.ItemType.VIDEO_CALL)
                     if (allMedias.contains(Engagement.MediaType.MESSAGING) && isAuthenticatedUseCase()) add(EntryWidgetContract.ItemType.SECURE_MESSAGE)
 
-                    if (allMedias.isEmpty()) {
+                    if (this.isEmpty()) {
```

Comment:
This solution looks great, but I think it might be more reliable to check the existing media items and authentication in combination because adding some extra items before this step that might not be related to the media types may break this logic.

Reply:
Okay. This concern should be covered by this fixup: 6426daffd45c8e4f647b2c9c0b14c0f2406bdf14.